src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -156,6 +168,10 @@ public abstract partial class JsonNamingPolicy
         public static System.Text.Json.JsonNamingPolicy CamelCase { get { throw null; } }
         public abstract string ConvertName(string name);
     }
+    public abstract partial class JsonNode
+    {
+        internal JsonNode() { }
```

Comment:
NIT: Wasn't this supposed to be private protected? @bartonjs

Reply:
It actually is in `JsonNode.cs`.

Reply:
how come that wasn't preserved here?

Reply:
It's just what `GenerateReferenceSource` does :confused: